app/models/requests/requestable_decorator.rb

```diff
@@ -84,7 +84,7 @@ def in_library_use_required?
     end
 
     def off_site?
-      recap? || annex? || item_at_clancy? || held_at_marquand_library?
+      recap? || recap_pf? || annex? || item_at_clancy? || held_at_marquand_library?
```

Comment:
I would assume recap? would be enough here.  Why the check for pf?  Any item in ReCAP is offsite.